### PR TITLE
Fix NaN when total = 0

### PR DESCRIPTION
**Pull request by michel-soares**
Fix Error: attribute d: Expected number, "M NaN NaN A 117.5 …". when max (total) = 0
Fill all complete because if total = 0 any value is completed (100%).

### CODE REVIEW

File: src/round-progress.service.ts [hunk 1]
```diff
@@ -57,8 +57,8 @@ export class RoundProgressService {
          elementRadius: number, isSemicircle = false): string {
 
     let value = Math.max(0, Math.min(current || 0, total));
-    let maxAngle = isSemicircle ? 180 : 359.9999;
-    let percentage = (value / total) * maxAngle;
+    let maxAngle = isSemicircle ? 180 : 359.9999;
```
Comment (crisbeto) [inline src/round-progress.service.ts]: Can you remove the trailing white space here?

**Comment by michel-soares** [comment]
hi Kristiyan ,

     How long we have this change in a npm release? We have multiple developers and we use the npm versions.
     There are any other way to every developers use this changes automatically?

Thank you,

**Comment by crisbeto** [comment]
Sorry, I forgot to do the release. I'll do it a bit later today.

**Comment by crisbeto** [comment]
Alright, it's published as release 1.1.1.

**Pull request merged**